openapi/googleapis.bigquery/Module.md

```diff
@@ -0,0 +1,41 @@
+## Overview
+This is a generated connector for [Google BigQuery API v2.0](https://cloud.google.com/bigquery/docs/reference/rest) OpenAPI specification.
+The BigQuery API provides access to create, manage, share and query data.
+
+## Prerequisites
+
+Before using this connector in your Ballerina application, complete the following:
+
+* Create a [Google account](https://accounts.google.com/signup)
+* Obtain tokens 
+    1. Follow [this link](https://developers.google.com/identity/protocols/oauth2)
+ 
+## Quickstart
+
+To use the Google BigQuery connector in your Ballerina application, update the .bal file as follows:
+
+### Step 1: Import connector
+First, import the `ballerinax/googleapis.bigquery` module into the Ballerina project.
+```ballerina
+import ballerinax/googleapis.bigquery;
+```
+
+### Step 2: Create a new connector instance
+Create a `bigquery:ClientConfig` with the OAuth2 tokens obtained, and initialize the connector with it. 
+```ballerina
+bigquery:ClientConfig clientConfig = {authConfig : {token: "<Your access token>"}};
+bigquery:Client baseClient = check new Client(clientConfig, serviceUrl = "https://bigquery.googleapis.com/bigquery/v2");
```

Comment:
Do we need to show the usage of `serviceUrl` here? since its default

Reply:
yes, true. we can remove the serviceUrl.